composables/use-error-handler.ts

```diff
@@ -34,12 +34,13 @@ export default function () {
     // Generic error handling
     if (!handler) {
       switch (rawErrorMessage) {
+        case 'INSUFFICIENT_PERMISSION':
         case 'TOKEN_EXPIRED':
           handler = {
             description: $t('error_token_expired'),
             onClose: () => {
               accountStore.logout()
```

Comment:
Do you need to await for this before `navigateTo`?

Reply:
Update with UI improvements